packages/apollo-gateway/src/__tests__/queryPlanCucumber.test.ts

```diff
@@ -49,10 +49,9 @@ features.forEach((feature) => {
               options
             );
 
-            const serializedPlan = JSON.parse(JSON.stringify(queryPlan, serializeQueryPlanNode));
```

Comment:
We're able to strip this out now, since the cucumber tests should use the real planner now :)